nuxeo-features/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/BatchUploadObject.java

```diff
@@ -104,6 +111,75 @@
 
     public static final String UPLOAD_TYPE_CHUNKED = "chunked";
 
+    private BatchManager batchManager;
+
+    public static class Constants {
+        public static final String MD5 = "md5";
+        public static final String MIMETYPE = "mimeType";
+        public static final String NAME = "name";
+        public static final String FILE_SIZE = "fileSize";
+        public static final String KEY = "key";
+    }
+
+    @Override protected void initialize(Object... args) {
+        super.initialize(args);
+        batchManager = Framework.getService(BatchManager.class);
```

Comment:
If you initialize the `BatchManager` here, please make sure it is used everywhere instead of calling `Framework.getService(BatchManager.class)` in the new and existing methods.

Reply:
Done